#  This message repressent a bar code data with a timestamp

Header header
string data
